src/Directory.Build.props

```diff
@@ -39,6 +39,14 @@
     <UpdateXlfOnBuild>true</UpdateXlfOnBuild>
     <ErrorOnOutOfDateXlf>true</ErrorOnOutOfDateXlf>
 
+    <UpdateXlfOnBuild Condition="'$(CIBuild)' != 'true'">true</UpdateXlfOnBuild>
+
+    <!-- Workaround that can be removed when we update machines to 15.6.
```

Comment:
This can only be removed if repo toolset sets this to true again. Is that the plan?

Reply:
Yes. Once on 15.6 i'll clean all this up.